### DISCUSSION TITLE
Some questions and how to contribute to the project

### DISCUSSION THREAD

**Discussion by jangwonhong** [the-qa-company/qEndpoint#424] [Q&A] [answered]
hello.
I read your paper and became very interested in this project.
I succeeded in uploading wikidata to my local pc according to the documentation you provided.
Thank you very much, I often write time consuming sparql queries on wikidata,
this project has helped me a lot.

1.
The only question I would like to ask is how to connect qendpoint to my wikibase.
Could you please provide a document that describes this in a little more detail?

2.
Also, I would like to contribute to this project as much as I have been helped by it.
The link to the "contribution guidelines" document seems to be broken.
If you can provide documentation, I would like to contribute to this project as much as I have been helped.

Thank you.

**Comment by D063520** [comment, marked as answer]
Hi @jangwonhong,

nice to meet you! About your questions:

1) did you try to follow https://github.com/the-qa-company/qEndpoint#connecting-with-your-wikibase? Where did you get stuck? Note you need to change the parameters and adapt them to your wikibase

2) I fixed the broken link! Thank you for the pointer. About contributing, what is your background? Do you have access to some infrastructure? Could you help for example maintaining indexes for some datasets that are use-full to the community?
 
Salut
Dennis

**Reply by jangwonhong** [reply]
Hi!

Thank you so much for pointing me to your comment!

1. I tried reinstalling it following the link you gave me, and it worked, thank you very much.
2. I'm not an experienced software engineer like you, but I do work with data engineering. You mentioned IT infrastructure as an example of where I can contribute, can you give me an idea of how many resources would be required to maintain the indexes for the datasets you are talking about? I'll talk to my teammates and see if we can help you with the resources we have available.

Wonhong
